typescript/cli/src/deploy/warp.ts

```diff
@@ -685,6 +700,38 @@ async function enrollRemoteRouters(
   return transactions;
 }
 
+/**
+ * Populates the destination gas amounts for each chain using warpConfig.gas OR querying other router's destinationGas
+ */
```

Comment:
I dont think we want to mix these 2 modes
`read` should query from onchain
`update` shouldnt rely on existing onchain values imo

Reply:
in this case, it has to. 

the new warp chain needs to know the other chain's destinationGas. the only way to do this is by reading existing onchain values